Gemfile

```diff
@@ -32,6 +32,7 @@ gem "net-pop" # needed for ruby upgrade to 3.1.0 https://www.ruby-lang.org/en/ne
 gem "net-smtp", require: false # needed for ruby upgrade to 3.1.0 for some dang reason
 gem "noticed" # Notifications
 gem "oj" # faster JSON parsing 🍊
+gem "pagy" # pagination
```

Comment:
gem seems legit, ok to add
https://rubygems.org/gems/pagy
https://github.com/ddnexus/pagy
![Screenshot 2025-02-03 at 7 54 58 AM](https://github.com/user-attachments/assets/476a5cf2-cb63-48da-999d-ee8bb618ceb4)